conf/defaults.ini

```
@@ -677,6 +677,7 @@ role_attribute_strict = false
 auto_sign_up = false
 url_login = false
 allow_assign_grafana_admin = false
+skip_org_role_sync = true
```

```suggestion
skip_org_role_sync = false
```

better not to change default behavior